bin/tbg

```diff
@@ -272,7 +272,15 @@ help()
 }
 
 #!/usr/bin/env bash
-initCall="$0 $*"
+#Check whether the provided path is relative or absolute and
+#convert it to an abolute path
+script="$0"
+case $script in
+        /*) absScriptPath=$script;; #called using abs. path
+        ./*) absScriptPath=$PWD/${script#"./"};; #called using ./, remove it
```

Comment:
There is no need for two three cases see sugestion: https://github.com/ComputationalRadiationPhysics/picongpu/issues/3189#issuecomment-591968591

Reply:
I don't think so. Consider the minimal working example:

    #!/bin/bash
    echo "$0"
    curpath=$0
    #convert to abs path
    case $curpath in
            /*) abspath=$curpath;;
           ./*) abspath=$PWD/${curpath#"./"};;
            *) abspath=$PWD/$curpath;;
    esac
    echo "Absolute path: "$abspath

Save it as a script as `realpathExample.sh` in a directory that is NOT in your `PATH`. Then using the suggestion from #3189 would result in an error and no absolute path. 

Removing the default case `*)` from the above will, for instance, prevent the user from obtaining the absolute path when running using `bash realpathExample.sh`.